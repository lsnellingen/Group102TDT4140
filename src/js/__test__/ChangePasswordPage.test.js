import React from 'react';
import ReactDOM from 'react-dom';
import ChangePasswordPage from '../pages/ChangePasswordPage';
import renderer from 'react-test-renderer';
import { RegistrationForm, LoginLink } from 'react-stormpath';
import {shallow} from 'enzyme';

describe('ChangePasswordPage', () => {
  var ReactTestUtils = require('react-addons-test-utils') // ES5 with npm

    test('renders without crashing', () => {
      /*const renderer = ReactTestUtils.createRenderer();*/

      const tree = renderer.create(
        <ChangePasswordPage location={{query: {sptoken:"ksdsl"}}}/>).toJSON();


      expect(tree).toMatchSnapshot();




    });
});
