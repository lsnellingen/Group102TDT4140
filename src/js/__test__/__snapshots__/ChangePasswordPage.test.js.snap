// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChangePasswordPage renders without crashing 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-xs-12"
    >
      <h3>
        Change Password
      </h3>
      <hr />
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col-xs-12"
    >
      <form
        onSubmit={[Function]}
      >
        <div>
          <div
            className="sp-change-password-form"
          >
            <div
              className="row"
            >
              <span />
              <div
                className="col-xs-12"
                data-spIf="!form.sent"
              >
                <div
                  className="form-horizontal"
                >
                  <div
                    className="form-group"
                  >
                    <label
                      className="col-xs-12 col-sm-4 control-label"
                      htmlFor="spPassword"
                    >
                      New Password
                    </label>
                    <div
                      className="col-xs-12 col-sm-4"
                    >
                      <input
                        className="form-control"
                        defaultValue=""
                        id="spPassword"
                        name="password"
                        onChange={[Function]}
                        placeholder="New Password"
                        required={true}
                        type="password"
                      />
                    </div>
                  </div>
                  <div
                    className="form-group"
                  >
                    <label
                      className="col-xs-12 col-sm-4 control-label"
                      htmlFor="spConfirmPassword"
                    >
                      Confirm New Password
                    </label>
                    <div
                      className="col-xs-12 col-sm-4"
                    >
                      <input
                        className="form-control"
                        defaultValue=""
                        id="spConfirmPassword"
                        name="confirmPassword"
                        onChange={[Function]}
                        placeholder="Confirm New Password"
                        required={true}
                        type="password"
                      />
                    </div>
                  </div>
                  <div
                    className="form-group"
                  >
                    <div
                      className="col-sm-offset-4 col-sm-4"
                    >
                      <span />
                      <button
                        className="btn btn-primary"
                        disabled={false}
                        type="submit"
                      >
                        <span />
                        <span
                          data-spIf="!form.processing"
                        >
                          Set New Password
                        </span>
                      </button>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </form>
    </div>
  </div>
</div>
`;
